typegate/native/src/runtimes/python/wasi_vm.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright Metatype OÜ, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+use std::path::PathBuf;
+use wasmedge_sdk::{
+    config::{CommonConfigOptions, ConfigBuilder, HostRegistrationConfigOptions},
+    error::HostFuncError,
+    host_function, params, Caller, ImportObjectBuilder, Module, Vm, VmBuilder, WasmValue,
+};
+
+#[host_function]
+pub fn callback(_caller: Caller, _args: Vec<WasmValue>) -> Result<Vec<WasmValue>, HostFuncError> {
+    // println!("[host] callback");
```

Comment:
fyi & async step, the callback will have to be used once Tokio is used within runtime